strings/extract_K_C_names.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+#2)Текстовый файл содержит записи о телефонах и их владельцах. Переписать в другой файл
+#телефоны тех владельцев, фамилии которых начинаются с букв К и С.
+
+def extract_k_c_names(input_file, output_file):
+    '''Function reads all entries from the input file and writes to the output file phones of the people which surnames start from "K" or "C". '''
+    
+    phones = []
+
+    if not os.path.exists(input_file):
+        raise FileNotFoundError('File does not exist.')
+
+    with open('entries.txt', 'r+') as file:
```

Comment:
It is better to use the 'input_file'  var here rather than specify the 'entries.txt' file

Reply:
Rectified